### ISSUE TITLE
After changing dimensions, inventory items dropped when clicked

### ISSUE THREAD

**Issue by KollinsPlays** [DarkPacks/SevTech-Ages#3220]
## Bug Report

Occasionally, after moving between dimensions (hunting dimension, betweenlands, twilight forest) the player inventory bugs out where whenever an item is clicked in either the hotbar, player or chest inventory, the item or item stack drops to the ground. Once this bug happens, it is persistent until not just unloading the world, but closing down sevtech and re-starting it. Some hotkeys work, like swap offhand, but clicking inventory items makes them drop on the ground.

## Expected Behaviour

items in inventory and chests should be able to be interacted with as normal ie left click, right click, pick up, move to chests.

## Steps to Reproduce (for bugs)

1. travel between dimensions (seems to be random and not tied to a specific dimension)
2. any attempt to interact with inventory using the mouse causes items to drop on the ground instead of the expected left click/right click/shift click action.

## Logs

* Client/Server Log:
* Crash Log:

## Client Information

* Modpack Version: 3.0.8
* Java Version: Version 8 update 171 build 1.8.0_171-b11
* Launcher Used: MultiMC
* Memory Allocated: 5023MB
* Server/LAN/Single Player: SSP
* Optifine Installed: No
* Shaders Enabled: No

## World Information

* Modpack Version world created in: 3.0.8

* Additional Content Installed: none

## Server Information
* Java Version:
* Operating System: Windows 7 x64
* Hoster/Hosting Solution:
* Sponge (Non-Vanilla Forge) Server:

**Comment by artdude543** [comment]
Duplicate of #2417

**Comment by KollinsPlays** [comment]
Saw the note that this may be a duplicate of #2417.
a) I have never seen this issue when first entering the world, only after playing and moving between dimensions.
b )Items in inventory show their names and do not show as "unfamiliar items"
c) I have some "unfamiliar items", mainly in age 3 just logs and saplings from some blue tree, that I was able to shift-click into a chest after picking them up having my hotbar filled with other stuff, but this bug drops items from any inventory slot. 
d) next time it happens, I will test with plant fiber, rocks and sticks.
